include/faker-cxx/Phone.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <optional>
+#include <string>
+
+namespace faker
+{
+class Phone
```

Comment:
please add a second number(PhoneNumberCountryFormat format) method with PhoneNumberCountryFormat enum with values like PhoneNumberCountryFormat::Italy, and with implementation add mapping from enum to concrete country formats from PhoneNumbers.h (you can check my code in Location module I did the setup for addresses).